<launch>
<node type="mocap_simulator_node.py" name="optitrack" pkg="me326_locobot_example" output="screen">
    <param name="frame_rate" value="0"/>
    <param name="publish_tf" value="true"/>
    <param name="fixed_frame_id" value="mocap"/>
    <rosparam param="model_list">[]</rosparam>
  </node>
</launch>